tooling/helium-server/worker/ssr.js

```diff
@@ -46,14 +46,30 @@ async function handleSsr(url, authToken = null, userAndAppearanceToken = null) {
     sha256,
     authToken
   );
-  const { httpResponse } = pageContext;
+
+  const { httpResponse, redirectTo } = pageContext;
+
+  let { statusCode, body } = httpResponse;
+  const headers = assembleHeaders(pageContext);
+
+  if (!statusCode) {
+    //set default if code not provided
+    statusCode = 302;
+  }
+
+  if (redirectTo) {
+    body = `Redirecting to ${redirectTo}.`;
+
+    return new Response(body, {
+      headers,
+      status: statusCode,
+      url: redirectTo
```

Comment:
Not sure `url` is a valid prop for the `options` obj when constructing a new `Response`. Any opposition to doing something like `return Response.redirect(redirectTo, 302)`?

Reply:
I have pushed this, but is would we need to make it a `new Response...`